app/furniture/journal/entry.rb

```diff
@@ -24,8 +24,11 @@ class Entry < ApplicationRecord
 
     # @!attribute journal
     #   @return [Journal::Journal]
-    belongs_to :journal, class_name: "Journal::Journal", inverse_of: :entries
-    delegate :room, :space, to: :journal
+    belongs_to :journal, inverse_of: :entries
+    has_one :room, through: :journal
+    has_one :space, through: :journal
```

Comment:
This was some minor cleanup from some poorly-connected linkages in the object graph. Happy to split this out.